_maps/shuttles/pirate_Interdyne.dmm

```diff
@@ -198,24 +200,24 @@
 /obj/effect/turf_decal/tile/dark_blue/opposingcorners,
 /obj/machinery/portable_atmospherics/canister/healium,
 /obj/machinery/atmospherics/components/unary/portables_connector/visible,
-/obj/item/wrench/medical,
 /turf/open/floor/iron/dark,
 /area/shuttle/pirate)
 "aJ" = (
 /obj/effect/turf_decal/tile/dark_blue/opposingcorners,
 /obj/machinery/computer/operating,
-/obj/item/disk/surgery/debug,
+/obj/item/disk/surgery/debug{
+	name = "Interdyne Surgical Operations Disk"
+	},
```

Comment:
I don't think players should have access to debug equipment regardless
You should make a new subtype surgery disk that adds the surgeries you explicitly want them to use

Reply:
I want them to be able to use all surgeries

Reply:
they already have this anyways im just giving the disk an appropriate name (which I did before idk why it didn't stay)

Reply:
> they already have this anyways
Yes, and I still don't think they should have debug gear, we shouldn't have to worry about people in the future adding new surgeries that are antithetical to these pirates

Explicitly debug gear shouldn't be handed out to players, you should be explicit rather than implicit with all the surgeries you give out.

Reply:
alright